frontend/src/components/Registration/Registration.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+
+import {
+  Button, 
+  CssBaseline,
+  Link,
+  Grid,
+  Typography,
+  Container,
+  FormHelperText,
+  Snackbar
+} from '@material-ui/core';
+import MuiAlert, { AlertProps }  from '@material-ui/lab/Alert';
+import StyledTextField from '../StyledTextField';
+import BaseService from '../../app/baseService';
+import useStyles from './Registration.style';
+
+export interface RegistrationProps {
+  
+};
```

Comment:
seems unnecesary (unless you expect there might be some changes in the future here)

Reply:
👍